cli/node/service.py

```diff
@@ -78,3 +78,13 @@ def get_initial_stake(self):
             return None, str(error)
 
         return node_initial_stake, None
+
+    def open(self):
+        """
+        Open the node to participate in the network.
+        """
+        open_node = loop.run_until_complete(self.service.node_management.open_node())
+
```

Comment:
Add exception, please.

Reply:
@anastasiia-bilova, done. But it couldn't be tested.